src/components/screens/wallet/overview/balanceInfo/index.js

```diff
@@ -13,7 +13,7 @@ import styles from './balanceInfo.css';
 
 // eslint-disable-next-line complexity
 const BalanceInfo = ({
-  t, activeToken, balance, isWalletRoute, address, username, isBanned, pomStart,
+  t, activeToken, balance, isWalletRoute, address, account, isBanned, pomStart,
```

Comment:
This component has evolved over time and it needs some clean up. Namely the parameters. now that you send the account instead of the username, you might as well remove the `balance`, `address`, `isBanned` and `pomStart`. Am I right?